dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -261,7 +261,7 @@ const frontsBannerAdStyles = css`
 
 const articleEndAdStyles = css`
 	position: relative;
-	min-height: ${getMinHeight(adSizes.outstreamDesktop.height)}px;
+	min-height: 450px;
```

Comment:
I think the `&.ad-slot-fluid` below is redundant now (sorry github is not letting me suggest!)